<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-11-04T11:57:17 IST" name="Default suite" finished-at="2023-11-04T11:57:17 IST" duration-ms="61">
    <groups>
    </groups>
    <test started-at="2023-11-04T11:57:17 IST" name="Default test" finished-at="2023-11-04T11:57:17 IST" duration-ms="61">
      <class name="testScripts.CreateUserTest">
        <test-method signature="createUserTest()[pri:0, instance:testScripts.CreateUserTest@6253c26]" started-at="2023-11-04T11:57:17 IST" name="createUserTest" finished-at="2023-11-04T11:57:17 IST" duration-ms="11" status="FAIL">
          <exception class="java.lang.Error">
            <message>
              <![CDATA[Unresolved compilation problems: 
home cannot be resolved
users cannot be resolved
excel cannot be resolved
addUser cannot be resolved
soft cannot be resolved
users cannot be resolved
course cannot be resolved
excel cannot be resolved
IConstantPath cannot be resolved to a variable
excel cannot be resolved
IConstantPath cannot be resolved to a variable
soft cannot be resolved
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Error: Unresolved compilation problems: 
home cannot be resolved
users cannot be resolved
excel cannot be resolved
addUser cannot be resolved
soft cannot be resolved
users cannot be resolved
course cannot be resolved
excel cannot be resolved
IConstantPath cannot be resolved to a variable
excel cannot be resolved
IConstantPath cannot be resolved to a variable
soft cannot be resolved

at testScripts.CreateUserTest.createUserTest(CreateUserTest.java:12)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Error -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createUserTest -->
      </class> <!-- testScripts.CreateUserTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
